packages/firestore/test/unit/specs/listen_spec.test.ts

```diff
@@ -1144,7 +1144,7 @@ describeSpec('Listens:', [], () => {
       .client(1)
       .stealPrimaryLease()
       .expectListen(query, 'resume-token-2000')
-      .watchAcksFull(query, 2000, docC)
+      .watchAcksFull(query, 3000, docC)
```

Comment:
This changed since we can use the read time to get a change diff, rather than a separate counter.